en/docs/references/ballerina-expression-syntax.md

```diff
@@ -37,6 +37,26 @@ To initialize a variable of type `float` in Choreo, you can use one of the follo
   +950.930
   ```
 
+### decimal
+
+The `decimal` type supports integer values with up to 34 decimal points and supports the same arithmetic operators as `int`. When entering literals, you can use `d` as the suffix.
+
+This type does not support `infinity`, `NaN`, or negative zero.
+
+The `decimal` type is a separate basic type and therefore counts as `anydata`.
```

Comment:
The fact that `decimal` is a separate basic type doesn't affect it being a subtype of `anydata`.

```suggestion
The `decimal` type is a separate basic type and is a subtype of `anydata`.
```